src/components/support/controllers.test.tsx

```diff
@@ -122,6 +122,34 @@ describe(controller.HandleSignupFormPost, () => {
     expect(response.body).toContain('We have received your request');
   });
 
+  it('should create a zendesk ticket correctly whilst signing user up for mailing list', async () => {
```

Comment:
This test name doesn't make sense. Do we really want to create a zendesk ticket? It looks like we sign them up to Mailchimp instead.

Reply:
The signup to mailchimp, is part of the sign up process. It's an extra box that can be ticked when requesting a paas account.

Reply:
OK. We probably need this feature available within the pazmin interface too. 🤔